e2e-playwright/utils/scope-helpers.ts

```diff
@@ -0,0 +1,294 @@
+import { Page, Response } from '@playwright/test';
+
+import { testScopes } from './scopes';
+
+const USE_LIVE_DATA = Boolean(process.env.API_CALLS_CONFIG_PATH);
+
+export type TestScope = {
+  name: string;
+  title: string;
+  children?: TestScope[];
+  filters?: Array<{ key: string; value: string; operator: string }>;
+  dashboardUid?: string;
+  dashboardTitle?: string;
+  disableMultiSelect?: boolean;
+  type?: string;
+  category?: string;
+  addLinks?: boolean;
+};
+
+type ScopeDashboardBinding = {
```

Comment:
Why not using https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/scopes.ts#L12 together with https://github.com/grafana/grafana/blob/main/public/app/features/apiserver/types.ts#L142 ? I'd avoid duplicating types that are alreadt defined.